src/server.c

```diff
@@ -1125,9 +1156,25 @@ struct nvnc* nvnc_open(const char* address, uint16_t port)
 	aml_unref(self->poll_handle);
 failure:
 	close(self->fd);
+	if (unix_) {
```

Comment:
What happens if someone tried to create a socket that already existed? It seems that this will cause the already existing socket to be unlinked.